client/mocks/builders_test.go

```diff
@@ -224,6 +226,30 @@ func buildCloudwatchLogsFiltersMock(t *testing.T, ctrl *gomock.Controller) clien
 	}
 }
 
+func buildConfigConfigurationRecordersMock(t *testing.T, ctrl *gomock.Controller) client.Services {
```

Comment:
I would move this to a new file, this is legacy testing code and we should slowly remove from it and depercate.